components/MailchimpForm.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/no-danger, no-dupe-keys */
+
+import React, { Component } from 'react';
+
+class MailchimpForm extends Component {
+  render() {
+    return (
+      <form
+        action="https://texasjusticeinitiative.us18.list-manage.com/subscribe/post?u=fd262cb4a5fc0bafb38da2e22&amp;id=2663621fac"
```

Comment:
I got this URL from logging into our Mailchimp account and copying it from the default "embedded form." It contains some parameters that are specific to our Mailchimp account, and I included them inline here since we'll be exposing them to our users anyway and it didn't seem like we'd gain much from tucking them away in environment variables.